ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/port-channel-short-esi.yml

```diff
@@ -7,9 +7,9 @@ servers:
   - name: server01_ESI
     rack: RackC
     adapters:
-      - endpoint_ports: [ Eth1 ]
-        switch_ports: [ Ethernet10 ]
-        switches: [ port-channel-short-esi ]
+      - endpoint_ports: [ Eth1, Eth2]
+        switch_ports: [ Ethernet10, Ethernet10]
+        switches: [ port-channel-short-esi, Leaf1]
```

Comment:
Why did you need to add the second element in the list here? (I am not sure I understand yet but maybe there is a good reason)

Reply:
Its because of the condition I have mentioned in the comment above -
https://github.com/aristanetworks/ansible-avd/pull/3027#discussion_r1266341984

Reply:
Lets add an error message instead of silently ignoring esi for single-homed devices.